orthopairs/src/test/java/org/reactome/release/orthopairs/OrthopairFileGeneratorTest.java

```diff
@@ -0,0 +1,81 @@
+package org.reactome.release.orthopairs;
+
+import org.json.simple.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({OrthopairFileGenerator.class, AlternateIdMapper.class})
+@PowerMockIgnore({"org.apache.logging.log4j.*", "javax.management.*", "javax.script.*",
+        "javax.xml.*", "com.sun.org.apache.xerces.*", "org.xml.sax.*", "com.sun.xml.*", "org.w3c.dom.*", "org.mockito.*"})
+public class OrthopairFileGeneratorTest {
+
+    private Map<String, Set<String>> mockMap = new HashMap<>();
+    private Set<String> mockSet = new HashSet<>();
+    private Map<String, Set<String>> mockAltIdMap = new HashMap<>();
+
+    @Mock
+    JSONObject mockJSONObject;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void createProteinHomologyFileTest() throws IOException {
+        String sourceTargetProteinMappingFilename = "sourceTargetProteinMappingFilename";
+        mockSet.add("UniProtKB=Q6DEX5");
+        mockSet.add("LDO");
+        mockMap.put("UniProtKB=Q86YI8", mockSet);
+        OrthopairFileGenerator.createProteinHomologyFile("speciesKey", sourceTargetProteinMappingFilename, mockJSONObject, mockMap);
```

Comment:
Does the file have contents after this line is run?  I'm still getting my head around mocking.  If so, it might be good to assert the file contains what you expect?

Reply:
I think JUnit has an assertion plugin for checking files easily